src/test/java/se/iths/jakartaeelab1/resource/PersonIT.java

```diff
@@ -0,0 +1,159 @@
+    }
+
+    @Test
+    @Order(2)
+    @DisplayName("addPersons should return response 201 and getAllPersons should return list with person")
+    void addPersonShouldReturnResponse201AndGetPersonsShouldReturnListWithPerson() {
+        RequestSpecification request = setUpRequest("{\"name\" : \"Kjell\",\"age\" : \"25\", \"profession\" : \"Kriminell\"}");
+
+        request.post().then().statusCode(201);
+        Persons persons = RestAssured.get("/persons").as(Persons.class);
+
+        assertThat(List.of(new PersonDto("Kjell", 25, "Kriminell"))).isEqualTo(persons.persons());
+    }
+    @Test
+    @Order(3)
+    @DisplayName("getPersonById should return person with that id")
+    void getPersonByIdShouldReturnPersonWithThatId() {
+        RequestSpecification request = setUpRequest("{\"name\" : \"Sten\",\"age\" : \"204\", \"profession\" : \"Landowner\"}");
+
+        UUID id = getUuidFromResponse(request.post());
+
+        PersonDto personDto = RestAssured.get("/persons/" + id).then()
+                .statusCode(200)
+                .extract()
+                .as(PersonDto.class);
+
+        assertThat(new PersonDto("Sten", 204, "Landowner")).isEqualTo(personDto);
+    }
+    @Test
+    @Order(4)
+    @DisplayName("updatePerson should return updated personDto with updated fields")
+    void updatePersonShouldReturnUpdatedPersonDtoWithUpdatedFields(){
+        RequestSpecification request = setUpRequest("{\"name\" : \"Ulla\",\"age\" : \"68\", \"profession\" : \"Gravedigger\"}");
+
+        UUID id = getUuidFromResponse(request.post());
+
+        String jsonString = "{\"name\" : \"Ulla\",\"age\" : \"70\", \"profession\" : \"Programmer\"}";
+        PersonDto personDto = RestAssured.given()
+                .contentType(ContentType.JSON)
+                .body(jsonString)
+                .when()
+                .patch("/persons/"+id)
+                .then()
+                .statusCode(200)
+                .extract()
+                .as(PersonDto.class);
+
+        assertThat(new PersonDto("Ulla",70,"Programmer")).isEqualTo(personDto);
+    }
+    @Test
+    @Order(5)
+    @DisplayName("deletePerson should remove person, getAllPersons should return list without that person")
+    void deletePersonShouldRemovePersonGetAllPersonsShouldReturnListWithoutThatPerson(){
+        RequestSpecification request = setUpRequest("{\"name\" : \"Remy Removal\",\"age\" : \"12\", \"profession\" : \"Cleaner\"}");
+
+        UUID id = getUuidFromResponse(request.post());
+
+        RestAssured.delete("/persons/" + id).then()
+                .statusCode(204);
+        Persons persons = RestAssured.get("/persons").as(Persons.class);
+        assertThat(persons.persons()).doesNotContain(new PersonDto("Remy Removal",12,"Cleaner"));
+    }
+    @Test
+    @Order(6)
+    @DisplayName("getPersonById with invalid Id should give status 404")
+    void getPersonByIdWithInvalidIdShouldGiveStatus404(){
+        RestAssured.get("/persons/" + 666)
```

Comment:
Suggesting to use a variable instead of '666' to clarify that its an invalid Id.

Reply:
Okay, good idea. Will do this :)